### PR TITLE
RDCC-1706, AAC 4.0: Bringing feature changes from Master to Demo

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-1706

### Change description ###

Bringing changes from: https://github.com/hmcts/rd-professional-api/pull/694

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/application-functional.yaml [hunk 1]
```diff
@@ -35,4 +35,7 @@ resendInterval: 60
 resendInviteEnabled : true
 
 # delete organisation enabled
-deleteOrganisationEnabled : ${DELETE_ORG:true}
\ No newline at end of file
+deleteOrganisationEnabled : false
```
Comment (KOTLAPRA23) [inline src/functionalTest/resources/application-functional.yaml:38]: why delete organisation disabling in the demo?

**Review by KOTLAPRA23 — CHANGES_REQUESTED** [review]
please do the required change

**Review by KOTLAPRA23 — DISMISSED** [review]

File: src/functionalTest/resources/application-functional.yaml (hunk 1 above)
Comment (AdilOoze) [reply src/functionalTest/resources/application-functional.yaml]: Changed by mistake, it has been corrected here: https://github.com/hmcts/rd-professional-api/pull/725/commits/37af4c1f1e0374a3746f2c4c71374d27734d7acb

**Review by KOTLAPRA23 — APPROVED** [review]

**Review by pksaxena01 — APPROVED** [review]
Approved

**Pull request merged**